doc/source/user_guide/categorical.rst

```diff
@@ -920,46 +923,6 @@ the resulting array will always be a plain ``Categorical``:
       # "b" is coded to 0 throughout, same as c1, different from c2
       c.codes
 
-.. _categorical.concat:
-
-Concatenation
-~~~~~~~~~~~~~
-
-This section describes concatenations specific to ``category`` dtype. See :ref:`Concatenating objects<merging.concat>` for general description.
-
-By default, ``Series`` or ``DataFrame`` concatenation which contains the same categories
-results in ``category`` dtype, otherwise results in ``object`` dtype.
-Use ``.astype`` or ``union_categoricals`` to get ``category`` result.
-
-.. ipython:: python
-
-   # same categories
-   s1 = pd.Series(['a', 'b'], dtype='category')
-   s2 = pd.Series(['a', 'b', 'a'], dtype='category')
-   pd.concat([s1, s2])
-
-   # different categories
-   s3 = pd.Series(['b', 'c'], dtype='category')
-   pd.concat([s1, s3])
-
-   pd.concat([s1, s3]).astype('category')
-   union_categoricals([s1.array, s3.array])
-
-
-Following table summarizes the results of ``Categoricals`` related concatenations.
```

Comment:
Can you keep this table? I think a good summary of what happens

Reply:
Though update to reflect current status

Reply:
I think it would be useful it it said a bit more about the return types, though it might be better to just point to type promotion docs for that. How about something like this?

```rst
+----------+--------------------------------------------------------+----------------------------+		
 | arg1     | arg2                            |      identical       | result                     |		
 +==========+========================================================+============================+		
 | category | category                        | True                 | category                   |		
 +----------+--------------------------------------------------------+----------------------------+		
 | category (object)| category (object)       | False                | object (dtype is inferred) |		
 +----------+--------------------------------------------------------+----------------------------+		
 | category (int) | category (float)          | False                | float (dtype is inferred) |		
 +----------+--------------------------------------------------------+----------------------------+		
```

Reply:
Seems logical. Not sure if we have better verbiage than saying `category (object)` to refer to the categories of the categorical; @TomAugspurger might have thoughts

Reply:
I think I'd like to keep it close to the repr, since that should make it easier to relate to practice

```python
>>> pd.Categorical(list("abc"))                                                                 
[a, b, c]
Categories (3, object): [a, b, c]
```

Reply:
Sorry for the delay @WillAyd, I thought we were waiting on @TomAugspurger. Just pushed an update. Hopefully I got the table right.